### PR TITLE
Preview html files from `defaultBucket` in iframes

### PR DESCRIPTION
**Pull request by nl0**

### CODE REVIEW

File: catalog/app/components/Preview/loaders/Html.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import { PreviewData } from '../types'
+
+import * as Text from './Text'
+import * as utils from './utils'
+
+export const detect = utils.extIn(['.htm', '.html'])
+
+const IFrameLoader = ({ handle, children }) => {
+  const sign = AWS.Signer.useS3Signer()
```
Comment (akarve) [inline catalog/app/components/Preview/loaders/Html.js]: we should note, if it's true, that unlike markdown and vega, we don't sign any URLs internal to the HTML. is that right?

Comment (nl0) [reply catalog/app/components/Preview/loaders/Html.js]: that's right

File: catalog/app/components/Preview/renderers/IFrame.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react'
+import { styled } from '@material-ui/styles'
+
+const IFrame = styled((props) => (
```
Comment (akarve) [inline catalog/app/components/Preview/renderers/IFrame.js]: Please add note on Content-Type `text/html`

Comment (nl0) [reply catalog/app/components/Preview/renderers/IFrame.js]: where should i put that?

**Pull request merged**